src/index.js

```diff
@@ -33,14 +33,14 @@ export default ({
     min,
     idleTimeoutMillis,
     testOnBorrow,
-    ...otherConfig,
+    ...otherConfig
```

Comment:
Hmh, does it break even with the --harmony flag on ? 
It seems that running npm run build on the original still leaves:

> create: () => _phantom2.default.create(...phantomArgs).then(instance => {

In the code, so either node is able to handle the spread operator in certain cases or that may break as well is that specific branch is ever taken.

As far as "by" hand implementation of spread&rest goes... well, rest is ok but spread is ugly as sin, unless I can find a better way. I made a push that should be compatible with whatever runtime can run the babel-ified code, however it adds the _objectWithoutProperties function to the code, so I'm sort of torn about how readable it is at this point.